---
layout: about
title: about
permalink: /
subtitle: Senior Scientist at <a href='https://www.novonordisk.com/science-and-technology/research-and-technology-centres/oxford-research-centre.html'>Novo Nordisk</a>. Oxford, United Kingdom.

profile:
  align: right
  image: prof_pic.jpg
  image_circular: false # crops the image to make it circular
  more_info: 

news: true  # includes a list of news items
latest_posts: true  # includes a list of the newest posts
selected_papers: true # includes a list of papers marked as "selected={true}"
social: true  # includes social icons at the bottom of the page
---

My goal is to find genetic predictors of disease that are useful for diagnosis, treatment, and understanding the disease's biology. Traditionally, complex diseases like Alzheimer's disease are studied through genome-wide association studies ([GWAS](https://en.wikipedia.org/wiki/Genome-wide_association_study)). However, conventional GWAS have failed to find a significant share of those genetic causes. I am interested in closing this gap by researching machine learning methods that extract more information from GWAS. Specifically, I superimpose GWAS information onto [biological networks](https://en.wikipedia.org/wiki/Biological_network). Then, I explore such networks considering both the statistical association of each genetic factor - obtained from GWAS - and the biological context - obtained from the network. However, there are many open questions about how to do this, for instance:

- How to exactly integrate GWAS information with networks?
- What is the right strategy to find truly associated genetic factors? Which underlying assumptions are warranted?
- What sources of information should we consider when building those networks?
- What is the suitable unit of analysis? The SNP? The gene? The pathway?

With my multidisciplinary background in machine learning, statistical genetics, and biotechnology, I believe I am in a privileged position to tackle such questions
